index.html

```diff
@@ -0,0 +1,184 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>TGMC 22</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <div class="header">
+        <a href=""><img width="50px" src="images/facebook.jpg" alt=""></a> 
+       <a href=""><img width="50px" src="images/twitter.png" alt=""></a> 
+        <div><a class="english" href="index.html">English</a></div>
+        <div><a href="" class="english">Homepage</a> </div>
+        <div><a href="" class="english"> Logout</a></div>
+    </div>
+    <header>
+        <div><a href=""><img class="logo" width="150px" src="images/drum-logo-design-template-vector-260nw-1594390288.webp" alt=""></a>
+        </div>
+        <nav>
+            <img class="hamburger" id="hamburger" width="40px" src="images/588a64e7d06f6719692a2d11.png" alt="">
+            <div class="hamburger-container">
+                <div class="close"><img width="30px" id="closeBtn"
+                        src="images/67-679412_x-cross-close-symbol-icon-button-gui-grey.png" alt=""></div>
+
+                <ul class="hamburger-items ">
+                    <li><a href="about.html" class="items" id="menuabout">About</a></li>
+                    <li><a href="" class="items" id="program">Program</a> </li>
+                    <li><a href="" class="items" id="join">Join</a></li>
+                    <li><a href="" class="items" id="Sponsor"> Sponsor</a></li>
+                    <li><a href="" class="items" id="enlist">Enlist</a> </li>
+                </ul>
+
+            </div>
+            <ul class="nav-items">
+                <li><a href="about.html" class="item" id="menuabout">About</a></li>
+                <li><a href="" class="item" id="program">Program</a> </li>
+                <li><a href="" class="item" id="join">Join</a></li>
+                <li><a href="" class="item" id="Sponsor"> Sponsor</a></li>
+                <li><a href="" class="item" id="enlist">Enlist</a> </li>
+            </ul>
+        </nav>
+    </header>
```

Comment:
- [ ] the black and the white navbars are bigger than the ones in the Design it would be better if you adjust the size of header in order to match the design 
- Yours 
![image](https://user-images.githubusercontent.com/61790587/175136818-3c66aa14-6633-42d6-a976-dfaf24de48b6.png)
- Design 
![image](https://user-images.githubusercontent.com/61790587/175136867-f2a82489-70fc-4445-9a88-a73162023db9.png)